### DISCUSSION TITLE
Is there a roadmap for Fooocus?

### DISCUSSION THREAD

**Discussion by ItsCrea** [lllyasviel/Fooocus#1830] [General]
It would be great to know about development plans and priorities.

**Comment by mashb1t** [comment]
The Fooocus dev team is currently fo(oo)cusing their attention elsewhere, see https://github.com/lllyasviel/Fooocus/blob/main/update_log.md and they will be back in mid/end of January.
I also hope that there then will be some kind of roadmap or project insights on which they are working on. Let's see!
